Sources/TSCUtility/Triple.swift

```diff
@@ -108,6 +108,11 @@ public struct Triple: Encodable, Equatable {
             return candidate
         }
 
+        // LLVM target triples support this alternate spelling as well.
+        if string.hasPrefix("macos") {
```

Comment:
would creating a dictionary based on OS.allCases and then adding an entry for "macOS" be cleaner?

Reply:
Yah, that seems reasonable to me, let me update.